diff --git a/node_modules/@clerk/shared/dist/react/index.mjs b/node_modules/@clerk/shared/dist/react/index.mjs
index f5c29c6..a943c55 100644
--- a/node_modules/@clerk/shared/dist/react/index.mjs
+++ b/node_modules/@clerk/shared/dist/react/index.mjs
@@ -979,7 +979,7 @@ var useCheckout = (options) => {
 
 // src/react/commerce.tsx
 import React6, { useCallback as useCallback4, useEffect as useEffect2, useMemo as useMemo3, useState as useState3 } from "react";
-import useSWR from "swr";
+import { useSWR } from "swr";
 import useSWRMutation from "swr/mutation";
 
 // src/react/stripe-react/index.tsx
